src/strawchemy/schema/scalars/base.py

```
@@ -58,3 +58,19 @@ def _serialize(value: timedelta) -> str:
     serialize=_serialize_date,
     parse_value=wrap_parser(datetime.fromisoformat, "DateTime"),
 )
+
+
+def _serialize_hstore(value: dict[str, str]) -> dict[str, str]:
+    return dict(value)
+
+
+def _parse_hstore(value: Any) -> dict[str, str]:
```

Ruff (0.15.2)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)